src/SprykerEco/Zed/Econda/Business/EcondaBusinessFactory.php

```diff
@@ -135,7 +127,7 @@ protected function createFileExporter()
         return new FileExporter(
             $this->createFileWriter(),
             new FailedResultTransfer(),
-            $this->createBatchResultModel(),
+            new BatchResultTransfer(),
```

Comment:
Ah now I see what the problem is: You now have 2 "new".
Why are you passing the transfer in here? I do not see a stateful object here as good constructor dependency.